tools/upgrade-loader-or-syncer-to-dm.md

```diff
@@ -0,0 +1,95 @@
+
+Loader is used to load the full data that is dumped from [mydumper](../tools/mydumper.md) to TiDB.
+
+When the `task-mode` of the task DM executes is `full`, DM automatically uses `dumper` to export data and then uses `loader` to import the data.
+
+To upgrade Loader to DM, you only need to generate the corresponding task configuration file and use dmctl to execute the task.
+
+### Change configuration 
+
+Loader uses the TOML file to define the process-related operation parameters and synchronization task parameters; DM uses the YAML file to define task configuration file parameters.
+
+Taking the configuration options in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md) as examples, the corresponding relationship of task configuration options between Loader and DM is as follows:
+
+| Configuration in Loader | Corresponding configuration in DM |
+| ---------------------- | --------------------------------- |
+| `pool-size` | `pool-size` of `loader` |
+| `dir` | `dir` of `loader` |
+| `db` | `target-database` |
+| `alternative-db` | Deprecated |
+| `source-db` | Deprecated |
+| `route-rules` | `route-rules` |
+| `pattern-schema` | `schema-pattern` |
+| `pattern-table` | `table-pattern` |
+| `do-db` |	`do-dbs` of `black-white-list`. The filtering feature of databases has been refactored. For detailed configuration, see `do-dbs` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |  
+| `do-table` | `do-tables` of `black-white-list`. The filtering feature of tables has been refactored. For detailed configuration, see `do-tables` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |  
+| `ignore-db` | `ignore-dbs` of `black-white-list`. The filtering feature of databases has been refactored. For detailed configuration, see `ignore-dbs` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |
+| `ignore-table` | `ignore-tables` of `black-white-list`. The filtering feature of tables has been refactored. For detailed configuration, see `ignore-tables` in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md). |
+| `rm-checkpoint` | Deprecated. A configuration option with a similar feature is `remove-meta`. |
+
+Taking the configuration options in [Data Migration Task Configuration File](../tools/dm-task-configuration-file-intro.md) as examples, the corresponding relationship of task configuration options between mydumper and DM is as follows:
+
+| Configuration in mydumper | Corresponding configuration in DM |
+| ---------------------- | --------------------------------- |
+| `host` | No corresponding option. Fixed when DM-worker is deployed. |
+| `port` | No corresponding option. Fixed when DM-worker is deployed. |
+| `user` | No corresponding option. Fixed when DM-worker is deployed. |
+| `password` | No corresponding option. Fixed when DM-worker is deployed. |
```

Comment:
Is "Fixed" the meaning we want to express? Perhaps "Configured" or "Confirmed" or some other words? @csuzhangxc @CaitinChen

Reply:
maybe "Configured" is better